client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -17,6 +17,12 @@
                             </li>
                         </ul>
                     </oui-tile-description>
+                    <oui-action-menu data-compact data-align="end">
+                        <oui-action-menu-item
+                            data-text="{{ ::'status_action_renew_management' | translate }}"
```

Comment:
`text` attribute is deprecated, prefer using a nested span 
```suggestion
  <oui-action-menu-item>
    <span data-translate="status_action_renew_management"></span>
  </oui-actio-menu-item>
```